string chainName
int32 space
path path
int32 axisMask
float32[] durations
bool isAbsolute
---
